c7n/resources/waf.py

```diff
@@ -48,3 +53,26 @@ class resource_type(TypeInfo):
         'describe': DescribeRegionalWaf,
         'config': ConfigSource
     }
+
+
+@resources.register('wafv2')
+class WAFV2(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = "wafv2"
+        enum_spec = ("list_web_acls", "WebACLs", None)
```

Comment:
afaics the list web acls does require a scope parameter. it can be statically associated as the third param in the enum spec (dictionary), however in this context it sounds like we would want to grab both scopes, and would need to override it on the describe source itself.

Reply:
that's correct. List-web-acls need scope parameter and it is different for resource types. I feel, setting to empty here and specifying the correct scope in resource specific calls would be better than overriding describe source.

Reply:
You are right. I have added a default parameter. Now, wafv2 will list acls for REGIONAL. Specific resources can override (for example cloudfront to use CLOUDFRONT)